README.md

```diff
@@ -178,13 +177,15 @@ The ccxt library currently supports the following 133 cryptocurrency exchange ma
 |[![zaif](https://user-images.githubusercontent.com/1294454/27766927-39ca2ada-5eeb-11e7-972f-1b4199518ca6.jpg)](https://zaif.jp)                                                              | zaif               | [Zaif](https://zaif.jp)                                                              | 1     | [API](https://techbureau-api-document.readthedocs.io/ja/latest/index.html)                       |                                                                                                                             | Japan                                   |
 |[![zb](https://user-images.githubusercontent.com/1294454/32859187-cd5214f0-ca5e-11e7-967d-96568e2e2bd1.jpg)](https://vip.zb.com/user/register?recommendCode=bn070u)                          | zb                 | [ZB](https://vip.zb.com/user/register?recommendCode=bn070u)                          | 1     | [API](https://www.zb.com/i/developer)                                                            |                                                                                                                             | China                                   |
 
-The list above is updated frequently, new crypto markets, altcoin exchanges, bug fixes, API endpoints are introduced and added on a regular basis. See the [Manual](https://github.com/ccxt/ccxt/wiki) for details. If you don't find a cryptocurrency exchange market in the list above and/or want another exchange to be added, post or send us a link to it by opening an issue here on GitHub or via email.
+The list above is updated frequently, new crypto markets, exchanges, bug fixes, and API endpoints are introduced on a regular basis. See the [Manual](https://github.com/ccxt/ccxt/wiki) for more details. If you can't find a cryptocurrency exchange in the list above and want it to be added, post a link to it by opening an issue here on GitHub or sending us an email.
```

Comment:
In this case, I think, it should end with:

> **post** a link to it by opening an issue here on GitHub **or sending** us an email.

↓

> **post** a link to it by opening an issue here on GitHub **or send** us an email.

?

Reply:
Yeah, it changes the meaning... as they don't "post links" by "sending us an email", they _either_ "post links" or "send us emails".